### DISCUSSION TITLE
Cannot not configure layout for app.blade.php

### DISCUSSION THREAD

**Discussion by NaungYeHtet** [filamentphp/filament#9507] [Help] [answered]
Here are my steps creating a layout for **database-notifications**

`php artisan notifications:table` 
`php artisan migrate`

Create `resources/views/components/layouts/app.blade.php`

Add `@livewire('database-notifications')` in `resources/views/components/layouts/app.blade.php`

Create `/resources/views/notifications/database-notifications-trigger.blade.php`

`/resources/views/notifications/database-notifications-trigger.blade.php` 
```
<button type="button">
    Notifications ({{ $unreadNotificationsCount }} unread)
</button>
```

`AppServiceProvider.php`

`DatabaseNotifications::trigger('notifications.database-notifications-trigger');`

But not showing notification icon at topBar. And mainly Filament is not using `resources/views/components/layouts/app.blade.php` as its main layout.

Actually I do not understand what blade layout this is referring to.

<img width="615" alt="image" src="https://github.com/filamentphp/filament/assets/49832780/3bb9f62e-a7c4-4a9c-b366-79a1b9d08424">

https://filamentphp.com/docs/3.x/notifications/database-notifications

**Comment by sinnbeck** [comment, marked as answer]
I think its built into the panel if that is what you are trying to do?

https://filamentphp.com/docs/3.x/panels/notifications#overview

```php
$panel
->databaseNotifications();
